components/availability/src/Availability.svelte

```diff
@@ -1037,23 +1054,29 @@
           }
 
           .available-calendars {
-            display: inline-block;
+            display: none;
             position: relative;
             z-index: 2;
-            display: none; // TODO: temporary, until we rework this to not collide w/ time ranges
+            float: right;
```

Comment:
😱

Reply:
Floats are only 99% bad! This is a 1% case :)